src/main.rs

```diff
@@ -176,6 +176,12 @@ fn main() {
                 .short("i")
                 .help("Ignore non-zero exit codes."),
         )
+        .arg(
+            Arg::with_name("capture-out")
+                .long("capture-out")
+                .short("c")
+                .help("Capture and display stdout and stderr."),
```

Comment:
Maybe "Capture and display the output of the commands (for debugging purposes)"?